.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "xbmc/addons/kodi-addon-dev-kit"]
+	path = xbmc/addons/kodi-addon-dev-kit
+	url = git://github.com/AlwinEsch/kodi-addon-dev-kit.git
```

Comment:
i thought the whole idea of the dev kit was to stick it in its own repository as you have done?

see https://github.com/fetzerch/xbmc/pull/2 

this is how it works in my fork. the kodi build directory supplies an add-on development environment. this environment is currently copied from the main kodi git, but it can as well come from an external git. you then point the add-on build to this build directory where it find the add-on devkit (-DCMAKE_PREFIX_PATH=&lt;kodi_build_dir&gt;/build. the add-on binaries can be mirrored in the kodi build tree, so you can immediately use the produced binary with that kodi (-DKODI_BUILD_DIR=&lt;kodi_build_dir&gt;).

this way, i simply execute 'make' in the add-on and voila.